docs/reference/indices/templates.asciidoc

```diff
@@ -41,7 +41,9 @@ PUT _template/template_1
 }
 --------------------------------------------------
 // CONSOLE
-// TESTSETUP
+// TEST[setup:name]
```

Comment:
I need to figure out how to fix Docs tests. At the moment they fail because of warnings

Reply:
Normally we would update the docs to not use deprecated functionality. Is that possible here?

Reply:
it would - actually it would be much easier if I just use a new pattern with 8 prefix. But should this be done across all the docs?
That change is to fix the `DocsClientYamlTestSuiteIT`.  The options  is to prefix with8, assert about warnings, disable warning checks alltogether?